components/builder-api/src/server/resources/channels.rs

```diff
@@ -764,7 +829,7 @@ fn do_get_channel_package(req: &HttpRequest,
                         return Err(Error::SerdeJson(e));
                     }
                 };
-                return Ok(pkg_json);
+                return Ok(serde_json::from_str(&pkg_json).unwrap());
```

Comment:
Could we return Ok(_p) here?  Or possibly move the return into the match?

Reply:
![](https://media.giphy.com/media/aTx5OMKR7FCGQ/giphy.gif)